### PR TITLE
Add DIY_2400_TX_Experimental_ESP_* targets for the configurator

### PR DESCRIPTION
**Pull request by jurgelenas**

### CODE REVIEW

File: src/targets/diy_2400.ini [hunk 1]
```diff
@@ -41,6 +41,12 @@ extends = env:DIY_2400_TX_DUPLETX_via_ETX
 [env:DIY_2400_TX_DUPLETX_via_WIFI]
 extends = env:DIY_2400_TX_DUPLETX_via_ETX
 
+[env:DIY_2400_TX_Experimental_ESP_via_UART]
+extends = env:Unified_ESP32_2400_TX_via_UART
+board_config = experimental.tx_2400.experiment1
+
+[env:DIY_2400_TX_Experimental_ESP_via_WIFI]
+extends = env:Unified_ESP32_2400_TX_UART
```
Comment (JyeSmith) [inline src/targets/diy_2400.ini:49]: Should be `DIY_2400_TX_Experimental_ESP_via_UART`

**Comment by CapnBry** [comment]
But `experimental.tx_2400.experiment1` doesn't exist in targets.json so doesn't this need to add that as well?

**Comment by jurgelenas** [comment]
> `experimental.tx_2400.experiment1` doesn't exist in targets.json so doesn't this need to add that as well?

It was added before my PR. So it is already there.

**Comment by CapnBry** [comment]
> It was added before my PR. So it is already there.

Oh (slaps forehead) I didn't notice this was for that branch. My bad! 😞

**Review by CapnBry — APPROVED** [review]

**Comment by pkendall64** [comment]
You know we shouldn't be using this branch any more as the features have all been merged into master!
The only thing missing from master is a target definition.

**Pull request merged**